README.md

```diff
@@ -1,20 +1,122 @@
+| Error Analysis | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+| Causal Analysis | Yes | No | No | Yes | No | Yes (max 5 features due to expensiveness) | No | No | No | No |
+| Counterfactual | Yes | Yes | No | Yes | No | Yes | No | No | No | No |
+
+This is all available via Python SDK or CLI.
+
+## Set Up
+In this section, we will go over the basic setup steps that you need in order to generate Responsible AI insights for your models from SDK v2, CLI v2 and visualize the generated Responsible AI insights in [AML studio](https://ml.azure.com/).
+
+### Create an Azure subscription
+Create an Azure workspace by using the [configuration notebook](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb)
+
+### Install the required packages
+In order to install `azureml-responsibleai` package you will need a python virtual environment. You can create a python virtual environment using `conda`.
+```c
+conda create -n azureml_env python=3.8
+activate azureml_env
+```
+
+After activating your environment, if this is your first time running the RAI Dashboard in private preview then continue to the [setup instructions](https://github.com/Azure/RAI-vNext-Preview/blob/mergePrivatePreview_MAD/docs/Setup.md) to do a one-time setup for your workspace.
+
+
+
+### Generating Responsibleai AI Dashboard insights
+Once you have created an Azure workspace and registered your components in the one-time setup above, you can create a Responsible AI dashboard via the CLI or SDK. Start here for `examples` [folder](examples) to get started.
+
+### Viewing your Responsible AI Dashboard in the AzureML studio portal
+After generating the Responsible AI insights, you can view them in your associated workspace in AzureML studio, under your model registry.
+
+![01](images/01_model_registry.png)
+1. Go to your model registry in your AzureML studio workspace
+2. Click on the model you've uploaded your Responsible AI insights for
+
+![02](images/02_model_details.png)
+3. Click on the tab for `Responsible AI dashboard (preview)` under your model details page
+
+![03](images/03_responsibleaitoolbox.png)
+4. Under the `Responsible AI dashboard (preview)` tab of your model details, you will see a list of your uploaded Responsible AI insights. You can upload more than one Responsible AI dashboards for each model. Each row represents one dashboard, with information on which components were uploaded to each dashboard (i.e. explanations, counterfactuals, etc).
+
+![04](images/04_dashboard.png)
+5. At anytime viewing the dashboard, if you wish to return to the model details page, click on `Back to model details`
+<ol type="A">
+  <li>You can view the dashboard insights for each component filtered down on a global cohort you specify. Hovering over the global cohort name will show the number of datapoints and filters in that cohort as a tooltip.</li>
+  <li>Switch which global cohort you are applying to the dashboard.</li>
```

Comment:
I could have missed some name changes, but isn't there one 'global cohort' at all times. Then the other cohorts are created as subsets of it?

Reply:
yes

Reply:
If you can make that more clear that would be great

Reply:
Mainly just zapped 'global'